src/client.rs

```diff
@@ -399,6 +431,53 @@ pub extern "C" fn rustls_client_session_is_handshaking(
     }
 }
 
+/// Return the TLS protocol version that has been negotiated. Before this
+/// has been decided during the handshake, this will return 0. Otherwise,
+/// the u16 version number as defined in the relevant RFC is returned.
+/// https://docs.rs/rustls/0.19.1/rustls/trait.Session.html#tymethod.get_protocol_version
+#[no_mangle]
+pub extern "C" fn rustls_client_session_get_protocol_version(
+    session: *const rustls_client_session,
+) -> u16 {
+    ffi_panic_boundary! {
+        let session: &ClientSession = try_ref_from_ptr!(session);
+        match session.get_protocol_version() {
+            Some(v) => v.get_u16(),
+            None => 0
+        }
+    }
+}
+
+/// Get the ALPN protocol that was negotiated, if any. Store a pointer to a
+/// borrowed buffer of bytes, and that buffer's len, in the output parameters.
+/// The borrow lives as long as the session.
+/// If the session is still handshaking, or no ALPN protocol was negotiated,
+/// store NULL and 0 in the output parameters.
```

Comment:
Nit: 
```suggestion
/// Get the ALPN protocol that was negotiated, if any. Stores a pointer to a
/// borrowed buffer of bytes, and that buffer's len, in the output parameters.
/// The borrow lives as long as the session.
/// If the session is still handshaking, or no ALPN protocol was negotiated,
/// stores NULL and 0 in the output parameters.
```
Seems like the more appropriate verb tense given other doccomments in the crate.